### PR TITLE
[FIX] base: typo

### PR DESCRIPTION
**Pull request by blaggacao**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
@blaggacao please rebase this against master. Typo fixes should always be done in master and not in stable versions :)
CC @mart-e

**Comment by blaggacao** [comment]
As non maintainer I can't rebase in here, I guess... But yeah, thanks for the reminder... Duly noted.

**Comment by Yenthe666** [comment]
Perhaps the easiest is to create a new PR against master. Just wanted to make sure you knew.

File: odoo/addons/base/module/module.py [hunk 1]
```diff
@@ -283,7 +283,7 @@ def _get_icon_image(self):
         ('GPL-3 or any later version', 'GPL-3 or later version'),
         ('AGPL-3', 'Affero GPL-3'),
         ('LGPL-3', 'LGPL Version 3'),
-        ('Other OSI approved licence', 'Other OSI Approved Licence'),
+        ('Other OSI approved license', 'Other OSI Approved License'),
```
Comment (mart-e) [inline odoo/addons/base/module/module.py]: Please change only the label, not the selection. We use that value in different integrations like the app store.

**Comment by mart-e** [comment]
Well "licence" is English but not US so ok to change it. You should be able to change the target branch and rebase, it is your fork you can modify it.

**Comment by mart-e** [comment]
I have done the rebase and change, should be good now

@robodoo r+

**Pull request closed without merging**